scss/tokens/_raw.scss

```diff
@@ -1,82 +1,407 @@
+$ouds-color-functional-malachite-100: #edfcf0 !default;
+// $ouds-color-functional-malachite-200: #c1f6ca !default;
+// $ouds-color-functional-malachite-300: #94f0a4 !default;
+// $ouds-color-functional-malachite-400: #67e97e !default;
+$ouds-color-functional-malachite-500: #3de35a !default;
+// $ouds-color-functional-malachite-600: #1ecd3c !default;
+// $ouds-color-functional-malachite-700: #17a02f !default;
+// $ouds-color-functional-malachite-800: #0e621d !default;
+$ouds-color-functional-malachite-900: #0a4715 !default;
+$ouds-color-functional-scarlet-100: #ffe5e6 !default;
+// $ouds-color-functional-scarlet-200: #ffb2b3 !default;
+// $ouds-color-functional-scarlet-300: #ff8081 !default;
+// $ouds-color-functional-scarlet-400: #ff4d4e !default;
+// $ouds-color-functional-scarlet-500: #ff1a1b !default;
+$ouds-color-functional-scarlet-600: #ea0305 !default;
+$ouds-color-functional-scarlet-700: #b20002 !default;
+$ouds-color-functional-scarlet-800: #800001 !default;
+$ouds-color-functional-scarlet-900: #4d0001 !default;
+$ouds-color-functional-sun-100: #fff7d6 !default;
+$ouds-color-functional-sun-200: #ffed99 !default;
+// $ouds-color-functional-sun-300: #ffe270 !default;
+// $ouds-color-functional-sun-400: #ffd73d !default;
+$ouds-color-functional-sun-500: #ffd000 !default;
+$ouds-color-functional-sun-600: #d6aa00 !default;
+// $ouds-color-functional-sun-700: #a38200 !default;
+// $ouds-color-functional-sun-800: #665100 !default;
+$ouds-color-functional-sun-900: #3d3100 !default;
+$ouds-color-functional-white: #fff !default;
+$ouds-color-transparent-black-0: rgba(0, 0, 0, 0) !default;
+// $ouds-color-transparent-black-100: rgba(0, 0, 0, .04) !default;
+// $ouds-color-transparent-black-200: rgba(0, 0, 0, .08) !default;
+$ouds-color-transparent-black-300: rgba(0, 0, 0, .16) !default;
+$ouds-color-transparent-black-400: rgba(0, 0, 0, .24) !default;
+$ouds-color-transparent-black-500: rgba(0, 0, 0, .32) !default;
+// $ouds-color-transparent-black-600: rgba(0, 0, 0, .48) !default;
+// $ouds-color-transparent-black-700: rgba(0, 0, 0, .64) !default;
+// $ouds-color-transparent-black-800: rgba(0, 0, 0, .88) !default;
+// $ouds-color-transparent-black-900: #000 !default;
+// $ouds-color-transparent-white-0: rgba(255, 255, 255, 0) !default;
+// $ouds-color-transparent-white-100: rgba(255, 255, 255, .04) !default;
+// $ouds-color-transparent-white-200: rgba(255, 255, 255, .08) !default;
+// $ouds-color-transparent-white-300: rgba(255, 255, 255, .16) !default;
+// $ouds-color-transparent-white-400: rgba(255, 255, 255, .24) !default;
+// $ouds-color-transparent-white-500: rgba(255, 255, 255, .32) !default;
+// $ouds-color-transparent-white-600: rgba(255, 255, 255, .48) !default;
+// $ouds-color-transparent-white-700: rgba(255, 255, 255, .64) !default;
+// $ouds-color-transparent-white-800: rgba(255, 255, 255, .88) !default;
+// $ouds-color-transparent-white-900: #fff !default;
+// $ouds-color-orange-50: #fff2e6 !default;
```

Comment:
Same here

Reply:
Here it's because it comes from two different files: some colors are from the common raw tokens, and "orange" + "warm-gray" from the Orange brand file.